certif/app/routes/authenticated/team.js

```diff
@@ -0,0 +1,11 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class AuthenticatedTeamRoute extends Route {
+  @service currentUser;
+
+  model() {
+    const certificationCenterId = this.currentUser.currentAllowedCertificationCenterAccess.id;
+    return this.store.findAll('member', { adapterOptions: { certificationCenterId } });
```

Comment:
question ember, le adapter `urlForFindAll` dans `certif/app/adapters/member.js` match avec le this.store.findAll  c'est bien ça ?

Reply:
Yes

Reply:
thanks !